src/transformStateWithClones.js

```diff
@@ -7,7 +7,42 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const ResultObject = [];
+
+  for (const operation of actions) {
+    const CurrentObject = {};
```

Comment:
```suggestion
    const currentState = Object.assign({}, state);
```
camelCase
move it out of loop